code/game/gamemodes/objective_items.dm

```diff
@@ -309,36 +389,57 @@
 	difficulty = 10
 	exists_on_map = TRUE
 
+/obj/item/pinpointer/nuke/add_stealing_item_objective()
+	GLOB.steal_item_handler.objectives_by_path[/obj/item/pinpointer/nuke] += src
+
 /datum/objective_item/special/aegun
 	name = "an advanced energy gun"
 	targetitem = /obj/item/gun/energy/e_gun/nuclear
 	difficulty = 10
 
+/obj/item/gun/energy/e_gun/nuclear/add_stealing_item_objective()
+	GLOB.steal_item_handler.objectives_by_path[/obj/item/gun/energy/e_gun/nuclear] += src
+
```

Comment:
almost all of these ones below seem unfeasible to do, as they are created midround, and not maploaded

Reply:
it's there just in case, They won't be considered anyways unless exists_on_map is set to true